projects/docs/past_projects_overview.md

```diff
@@ -0,0 +1,14 @@
+# Past projects
+
+We are proud to showcase the project presentations our students prepared in previous iterations of *Computational Tools for Climate Science*. 
+
+You can browse their presentation slides by project topic:
+- [Sea level rise](../past-projects/Sea_level_rise.ipynb)
```

Comment:
same here for `projects/past-projects` I would guess

Reply:
Same reply as above :)

Reply:
same reply as above ^^